vllm/utils.py

```diff
@@ -628,14 +628,34 @@ def is_valid_ipv6_address(address: str) -> bool:
         return False
 
 
+def split_ip_port(ip_port: str) -> Tuple[str, int]:
```

Comment:
using `host` here instead of `ip` would be more consistent with your variable names. The function doesn't require it to be IP addresses, so host is a little more accurate

Reply:
@russellb PTAL.